src/env.js

```diff
@@ -155,6 +155,11 @@ const envDefinitions = {
     parser: boolParser,
     desc: "includes HTTP response body in logs",
   },
+  DEBUG_HTTP_RESPONSE_MAX_SIZE: {
+    def: 256,
```

Comment:
That's too small, i think we want to get rid of the extreme payload here.
Threshold can be something like 100k